webapp/src/components/Controls/Controls.tsx

```diff
@@ -129,6 +139,16 @@ const Controls: React.FC<Props> = ({
   const handleDatasetSplitChange = (name: DatasetSplitName) =>
     history.push(`/${jobId}/dataset_splits/${name}/${mainView}${searchString}`);
 
+  const handlePostProcessingToggle = (enable: boolean) =>
```

Comment:
Here, one capital `P` left. Also, I would change `Toggle` to `Change` to make it uniform with the other handlers. It would also avoid any possibility of confusion with a "toggle" button, which is a different component, but that's not the main reason.
```suggestion
  const handlePostprocessingChange = (enable: boolean) =>
```